hack/kind-cluster-v1.12.8.sh

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env bash
+
+k8s_version=v1.12.8
+clusterName=${1:-kind}
+
+echo "############ install kind ##############"
+if hash kind 2>/dev/null;then
+    echo "kind have installed"
+else
+    echo "start install kind"
+    GO111MODULE="on" go get sigs.k8s.io/kind@v0.4.0
+    echo PATH=\$PATH:$(go env GOPATH)/bin >> /etc/profile && source /etc/profile
+fi
+
+echo "############# start create cluster:[${clusterName}] #############"
+
+data_dir=${HOME}/kind/${clusterName}/data
+
+echo "clean data dir: ${data_dir}"
+if [ -d ${data_dir} ]; then
+    rm -rf ${data_dir}
+fi
+
+echo "init the mount dirs for kind cluster"
+mkdir -p ${data_dir}/worker{,1,2,3,4,5}/vol{1..9}
+
+cat <<EOF > kind-config.yaml
```

Comment:
one more nit: this `kind-config.yaml` file can be stored at `${HOME}/kind/${clusterName}/kind-config.yaml`
user can destroy all state together.